docs/installation/advanced-installation-options.rst

```diff
@@ -51,3 +51,37 @@ install the newer version of ``presto-admin`` by following the instructions in
 the installation section (see :ref:`presto-admin-installation-label`). Your
 configuration files located in ``/etc/opt/prestoadmin`` will remain intact and
 continue to be used by the newer version of ``presto-admin``.
+
+Coordinator failover
+--------------------
+Presto does not yet support automatic failover for the coordinator. But you can
+migrate to a new coordinator using ``presto-admin`` -H and -x flags to,
+respectively, exclude and include hosts in your command.
+
+To view these ``presto-admin`` options, use the ``--extended-help`` flag.
+
+You can switch to a new coordinator by following the steps below:
+
+1. Shutdown the cluster
+
+2. Edit the ``presto-admin`` topology file and replace the old coordinator
+with the new one.  By default, the topology file is located at
+``/etc/opt/prestoadmin/config.json``.
+
+3. To install Presto on the new node, run the following two ``presto-admin``
+commands. The first command is needed only if Java 8 is not already installed
+on the new coordinator:
+::
+
+    sudo ./presto-admin package install -H new_coordinator /path/to/jdk8.rpm
+    sudo ./presto-admin server install -H new_coordinator /path/to/presto-server.rpm
+
+4. Update the coordinator and worker configuration files controlled by
+presto-admin. By default, these files are available at ``/etc/opt/prestoadmin/``.
```

Comment:
Format presto-admin to be wrapped by two backticks since that's how it's formatted every where else.